src/guide/jbs-jdk-bug-system.md

```diff
@@ -81,7 +81,7 @@ The [Affects Version/s]{.jbs-field} field is used to indicate which releases an
 
 1) If an issue is applicable to release N, then it can't be assumed that it is applicable to older releases less than N. It may be, but in general this is less important to know, as the majority of issues are only fixed in the latest feature release. If the issue is a crash or important in another way, then it becomes worthwhile to take the time to determine if it's relevant to earlier LTS releases.
 
-Another aspect of an issue is when the feature it's a part of was added or removed from the JDK, which in either case limits the range of releases the issue impacts. Knowing that a feature was removed before the oldest currently maintained release means it can be resolved as [Won't Fix]{.jbs-value}.
+Another aspect of an issue is when the feature, it's a part of, was added or removed from the JDK, which in either case limits the range of releases the issue impacts. Knowing that a feature was removed before the oldest currently maintained release means it can be resolved as [Won't Fix]{.jbs-value}.
```

Comment:
Maybe "... when a part of the feature was added ...".  I'm not sure because that may change the intent.

Reply:
This could be more generalized since it doesn't only hold for features, but basically any code.
"...is when the impacted code was added or..."